addon/templates/components/ilios-calendar.hbs

```diff
@@ -11,11 +11,12 @@
 </ul>
 <div class='ilios-calendar-calendar'>
   <h1 class="{{if calendarEventsPromise.isFulfilled 'loaded'}}">{{fa-icon 'spinner' spin=true}} {{loadingMessage}}</h1>
-  {{component (concat "ilios-calendar-" selectedView) 
+  {{component (concat "ilios-calendar-" selectedView)
     calendarEvents=calendarEventsPromise.content
     date=selectedDate
     selectEvent='selectEvent'
     changeDate='changeDate'
     changeView='changeView'
+    dueThisDay=dueThisDay
```

Comment:
Like the other properties the addon should default to english by setting the plain value.  Then the one level higher can replace that with a translation if necessary.  This is going to be important because when we deploy this calendar in a smaller focused app for our students it will only be in english.